ci/envs/312-latest-conda-forge_no_pyproj.yaml

```diff
@@ -6,7 +6,7 @@ dependencies:
   # required
   - pandas
   - shapely
-  - fiona
+  # - pyogrio
```

Comment:
Yes

Reply:
Does it actually more than fiona? I think for pyogrio, pyproj is only an optional dependency that is used in case of geopandas input for writing, and similarly for fiona in our `_to_file_fiona` we raise an error if pyproj is not installed?

Reply:
Ah, but I assume our tests still pass with fiona because if pyproj is not available the `crs` will always be None, while in pyogrio we always import pyproj in the geopandas code path, regardless of whether `crs` is set.

Reply:
Yes. To have a working env without pyproj, we need IO engine that does not require it, so Fiona. With pyogrio, we can't use IO without pyproj. I don't think that is an issue as the idea behind pyproj being optional is to depend on pandas+shapely only. In any case, this env needs to have Fiona and not pyogrio.